packages/cli/src/commands/kill.js

```diff
@@ -2,8 +2,9 @@
 import os from 'os';
 import { E } from '@endo/far';
 import { withEndoParty } from '../context.js';
+import { parsePetNamePath } from '../pet-name.js';
 
 export const killCommand = async ({ name, partyNames }) =>
   withEndoParty(partyNames, { os, process }, async ({ party }) => {
-    await E(party).terminate(name);
+    await E(party).terminate(parsePetNamePath(name));
```

Comment:
can we just support paths natively in `terminate`

Reply:
In my understanding, the decision was to only make the "hub" (i.e. agent (guest / host) and, in the future, directories) `lookup` variadic.

I could change the way we call `parsePetNamePath` such that its part of how `commander` coerces options / arguments, though. (See e.g. usage of `collect` in the CLI `endo.js`.)

Reply:
Parsing pet name paths must be a UI (CLI or GUI) concern because they’re likely to handle that concern differently in the long run. GUI will be in a position to support other kinds of tokenization.